src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -5,12 +5,16 @@
 import com.revrobotics.CANSparkBase.IdleMode;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Servo;
 import frc.robot.constants.LiftConstants;
 
 public final class Lift extends SubsystemBase {
     private CANSparkMax leftLiftMotor = new CANSparkMax(LiftConstants.leftMotorPort, MotorType.kBrushless);
     private CANSparkMax rightLiftMotor = new CANSparkMax(LiftConstants.rightMotorPort, MotorType.kBrushless);
 
+    private Servo leftServo = new Servo(LiftConstants.leftServoPort);
+    private Servo rightServo = new Servo(LiftConstants.rightServoPort);
```

Comment:
Maybe rename to "leftRatchetServo" and "rightRatchetServo"

Reply:
I would agree with this. More descriptive is usually better unless it gets excessive